papers/emily_dorne/main.md

```diff
@@ -15,31 +15,35 @@ abstract: |
 
-CyFi, short for Cyanobacteria Finder, is an open-source Python package that uses satellite imagery and machine learning to detect cyanobacteria levels. CyFi helps decision makers protect the public by flagging the highest-risk areas in lakes, reservoirs, and rivers quickly and easily. CyFi represents a significant advancement in environmental monitoring, providing higher resolution detection capabilities that can pinpoint areas at risk of cyanobacterial contamination. The strength of CyFi lies in its utilization of Sentinel-2 satellite data, a computationally efficient gradient boosted tree machine learning algorithm, and a straightfoward command line interface.
+[**CyFi**](https://cyfi.drivendata.org/)**, short for Cyanobacteria Finder, is an open-source Python package that uses satellite imagery and machine learning to detect cyanobacteria levels.** Cyanobacteria, or blue-green algae, is a major type of HAB. CyFi helps decision makers protect the public by flagging the highest-risk areas in lakes, reservoirs, and rivers quickly and easily. CyFi represents a significant advancement in environmental monitoring, providing higher-resolution detection capabilities that can pinpoint areas at risk of cyanobacterial contamination. Key strengths of CyFi compared to other tools include:
+- Features derived from high-resolution Sentinel-2 satellite data
+- A fast and computationally efficient boosted tree machine learning algorithm
+- A straightforward command line interface
+- A new training dataset of almost 13,000 data points across the continental U.S.
 
-This paper presents a detailed examination of the development of CyFi, exploring the algorithm's evolution, the dynamics of the prize competition that shaped it, and the broader implications of such competitions for scientific advancements. 
+This paper presents a detailed examination of the development of CyFi. It will explore the algorithm's evolution, learnings about what model-building methods are most effective for inland HABs, and the broader implications of machine learning competitions for scientific advancements. CyFi was originally developed as part of an open-souce machine learning competitions. Takeaways from the winning models were combined during a model experimentation phase, and shaped by real-world context surfaced through user interviews with water quality managers.
```

Comment:
```suggestion
This paper presents a detailed examination of the development of CyFi. It will explore the algorithm's evolution, learnings about what model-building methods are most effective for inland HABs, and the broader implications of machine learning competitions for scientific advancements. CyFi was originally developed as part of an open-souce machine learning competition. Takeaways from the winning models were combined during a model experimentation phase, and shaped by real-world context surfaced through user interviews with water quality managers.
```